### PR TITLE
SKIL-428-2

### PR DESCRIPTION
**Pull request by aparriaran**
TODOs Completed:
-Created Jest Tests for ViewAssessmentTask
-Added conditional to aria-label for EditAssessmentTask
-Updated the aria-label names from completeAssessmentTaskButton to startAssessmentTaskButton

### CODE REVIEW

File: FrontEndReact/src/View/Student/__tests__/TADashboard.test.js
Comment (ebanderson3) [inline FrontEndReact/src/View/Student/__tests__/TADashboard.test.js:1]: Is this supposed to be `satb`?

**Review by ebanderson3 — APPROVED** [review]
@sah0017 LGTM!

**Review by jcallison1 — APPROVED** [review]
@sah0017 LGTM

**Comment by aparriaran** [comment]
@sah0017 I am going to merge this PR!

**Pull request merged**